content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -363,6 +363,122 @@ to configure the `service` tag only in the configuration of the process check.
 {{% /tab %}}
 {{< /tabs >}}
 
+### Serverless environment
+
+#### AWS Lambda Functions 
+
+Depending on how you build and deploy your AWS Lambda-based serverless applications, you may have several options available for applying the `env`, `service` and `version` tags to metrics, traces and logs.
+
+Note, these tags are expected to be specified through AWS resource tags, instead of environment variables. Specifically, the `DD_ENV`, `DD_SERVICE` and `DD_VERSION` environment variables are not supported.
+
+{{< tabs >}}
+
+{{% tab "Serverless Framework" %}}
+
+Tag your Lambda functions using the [tags][1] option:
+
+```yaml
+# serverless.yml
+service: service-name
+provider:
+  name: aws
+  # to apply the tags to all functions
+  tags:
+    env: "<ENV>"
+    service: "<SERVICE>"
+    version: "<VERSION>"
+
+functions:
+  hello:
+    # this function will inherit the service level tags config above
+    handler: handler.hello
+  world:
+    # this function will overwrite the tags
+    handler: handler.users
+    tags:
+      env: "<ENV>"
+      service: "<SERVICE>"
+      version: "<VERSION>"
+```
+
+If you have installed the [Datadog serverless plugin][2], the plugin automatically tag the Lambda functions with the `service` and `env` tags using the `service` and `stage` values from the serverless application definition, unless a `service` or `env` tag already exists.
+
+[1]: https://www.serverless.com/framework/docs/providers/aws/guide/functions#tags
+[2]: https://docs.datadoghq.com/serverless/serverless_integrations/plugin
+{{% /tab %}}
+
+{{% tab "AWS SAM" %}}
+
+Tag your Lambda functions using the [Tags][1] option:
+
+```yaml
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Resources:
+  MyLambdaFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Tags:
+        env: "<ENV>"
+        service: "<SERVICE>"
+        version: "<VERSION>"
+```
+
+If you have installed the [Datadog serverless macro][2], you can also specify a `service` and `env` tag as parameters:
+
+```yaml
+Transform:
+  - AWS::Serverless-2016-10-31
+  - Name: DatadogServerless
+    Parameters:
```

Comment:
Are we adding functionality to support adding tags on all functions in your `template.yml`? Without additional context, this looks like a typo.

Reply:
Isn't this supported? https://github.com/DataDog/datadog-cloudformation-macro/tree/master/serverless#aws-sam Did I miss anything?

Reply:
I meant specifically for the version tag since that's not listed in the example here

Reply:
Yeah, I think we need to update the macro to support the version tag first, and then come back to update this doc, since the version tag is not yet supported by the macro?